api-docs/rst/dev-guide/concepts.rst

```diff
@@ -111,6 +111,16 @@ The launch configuration specifies the launch type along with server and load ba
              Not used when there is no ``loadbalancers`` configuration. Please note that
              this feature only works with a cloud load balancer.
 
+Each scaling group has an associated ``status`` that represents the health of the
+group. When the group is successfully able to launch servers and optionally add
+them to load balancers then the status is ``ACTIVE``. If it is not able to
+that due to some irrecoverable error that requires user attention then status
```

Comment:
An *ACTIVE* status indicates that the group can launch servers successfully and fulfill any requests to add servers to load balancers.  An *ERROR* status indicates an issue with scaling, policy, or convergence activities that requires attention.